doc/services/pm/device.rst

```diff
@@ -40,7 +40,7 @@ For more information, see :ref:`pm-device-runtime`.
 System Power Management
 ***********************
```

Comment:
While you're at, it would be good to disambiguate System Power Management naming.
There's already a `System Power Management` defined [here](https://docs.zephyrproject.org/latest/services/pm/system.html)

Reply:
My bad. Good move!